wp-content/themes/missilethreat/assets/_scss/abstracts/_mixins.scss

```diff
@@ -48,6 +48,16 @@
   font-size: rem($size);
 }
 
+//
+@mixin font-clamp($min, $max) {
+  $min-rem: rem($min);
+  $half-min-rem: calc(#{$min-rem} / 2);
+  font-size: $min; //Fallback in px
+  font-size: $min-rem;
+  // font-size: #{'clamp(#{rem($min)}, (#{$value} + 0.333vw), #{rem($max)})'};
```

Comment:
Let's remove this line. Generally speaking, unless there's a reason to keep it, we should remove commented out lines of code.